### PR TITLE
Add missing parameter to "Add user to team" PHP code sample

### LINKED ISSUE #552: Wrong example for Add user to team with PHP Driver
The example for [Add user to team](https://api.mattermost.com/#tag/teams/paths/~1teams~1{team_id}~1members/post) for the PHP Driver does not work.
The given example is:
```php
$resp = $driver->getTeamModel()->addUser($teamID, [
  "user_id" => $userID,
]);
```
which fails with the response:
```json
{
  "id": "api.context.invalid_body_param.app_error.",
  "message": "team_id invalid or missing in request body.",
  "detailed_error": "",
  "request_id": "xxxxxxxxxxxxxxxxxxxxxxxxxx",
  "status_code": 400
}
```

With this request in PHP, it works:
```php
$resp = $driver->getTeamModel()->addUser($teamID, [
  "user_id" => $userID,
  "team_id" => $teamID,
]);
```

### PR DESCRIPTION
**Pull request by nathanaelhoun**
#### Ticket Link

Fixes #552

### CODE REVIEW

File: v4/source/teams.yaml [hunk 1]
```diff
@@ -1145,6 +1145,7 @@
 
             $resp = $driver->getTeamModel()->addUser($teamID, [
```
Comment (larkox) [inline v4/source/teams.yaml:1146]: Is it needed to pass the team id twice? It is being passed here already.

Comment (nathanaelhoun) [reply v4/source/teams.yaml:1162]: I agree, this does not seem very useful. But the PHP API works like this for the moment, see the details in the linked issue (https://github.com/mattermost/mattermost-api-reference/issues/552)

Comment (larkox) [reply v4/source/teams.yaml:1162]: Thanks! Sorry for not checking the issue previously :sweat_smile:

**Comment by jasonblais** [comment]
Thank you @nathanaelhoun for the contribution :)

**Pull request merged**